src/image/Image.cc

```diff
@@ -743,7 +736,7 @@ bool imagesOverlap(ImageBase<T1> const& image1, ImageBase<T2> const& image2) {
     INSTANTIATE_OPERATOR(*=, T); \
     INSTANTIATE_OPERATOR(/=, T)
 
-#define INSTANTIATE2(T1, T2) template bool imagesOverlap<T1, T2>(ImageBase<T1> const&, ImageBase<T2> const&);
+#define INSTANTIATE2(T1, T2) template bool imagesOverlap<T1, T2>(ImageBase<T1> const&, ImageBase<T2> const&)
```

Comment:
Is this a good idea? IIRC omitting a semicolon for macro definitions confuses Doxygen.